### DISCUSSION TITLE
Treat `pnpm-lock.yaml` `yarn.lock` as generated files by default

### DISCUSSION THREAD

**Discussion by typed-sigterm** [community/community#111223] [Repositories]
### Select Topic Area

Product Feedback

### Body

Just like with npm's `package-lock.json`, both `pnpm-lock.yaml` and `yarn.lock` are Node.js lockfiles used widely (pnpm: 253k code, yarn: 373k code, on GitHub). However, only `package-lock.json` is excluded from diff views. Although we can  [opt-in manually](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github), an out-of-box support would be cooler :)

**Comment by spenserblack** [comment]
Related: https://github.com/github-linguist/linguist/issues/4348

**Comment by typed-sigterm** [comment]
Solved by github-linguist/linguist#6749 in v7.29.0. It works now.
